test/xref2/module_type_of_size.t/gen.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
I guess this file wasn't meant to be there ? Otherwise, `compile.sh` and `odoc.sh` shouldn't be included.

Reply:
After all, I find `gen.sh` more readable than the full test.
It could even be inlined in `run.t`, generating `compile.sh` and `odoc.sh` wouldn't be necessary as commands could be called directly in the loop.

Reply:
It's there in case we ever wanted to increase the size of the test, or modify the contents of the mli files. I wrote it to do this test so I thought I might as well include it :-)